src/ert/parsing/config_keywords.py

```diff
@@ -167,55 +176,83 @@ class SchemaItem(BaseModel):
     # Allowed values for specific arguments, if no entry, all values allowed
     indexed_selection_set: Mapping[int, List[str]] = {}
 
-    def check_valid(self, val: "FileContextToken", index: int) -> None:
-        if (
+    def _is_in_allowed_values_for_kw(
+        self, token: "FileContextToken", index: int
+    ) -> None:
```

Comment:
This is typed as having no return value but actually returns a boolean

Reply:
Oops, fixed